src/faction_camp.cpp

```diff
@@ -2278,6 +2278,9 @@ void basecamp::start_crafting( const std::string &cur_id, const point &cur_dir,
             components.consume_components();
             for( const item &results : making.create_results( batch_size ) ) {
                 comp->companion_mission_inv.add_item( results );
+                for (const item& byproducts : making.create_byproducts()) {
+                    comp->companion_mission_inv.add_item(byproducts);
+            }
```

Comment:
```suggestion
                for( const item &byproducts : making.create_byproducts() ) {
                    comp->companion_mission_inv.add_item( byproducts );
                }
```

Reply:
astyle requires a parameter with the astyle options. `astyle --options=.astylerc file`. Alternatively you can run `make astyle` and it will run astyle on all the game files. That only applies for linux and maybe mac. I don't know how it works on Windows.

Reply:
On windows you can run make astyle if you compile with make (there are a couple of ways to do that). If you use VS, there is a config file for the astyle extension, and you can use that.